backend/utils/llm.py

```diff
@@ -74,12 +74,12 @@ def should_discard_memory(transcript: str) -> bool:
 def get_transcript_structure(transcript: str, started_at: datetime, language_code: str, tz: str) -> Structured:
     prompt = ChatPromptTemplate.from_messages([(
         'system',
-        '''Your task is to provide structure and clarity to the recording transcription of a conversation.
+        '''You are an expert conversation analyzer. Your task is to analyze the conversation and provide structure and clarity to the recording transcription of a conversation.
```

Comment:
why we need to update the prompt?

Reply:
The previous prompt would also include a lot of items that the user might have already done, and it sometimes also includes items that are not necessarily actionable. The updated prompt forces it to make sure that the items are concise, actionable and are something that are not already completed by the user

Reply:
Ref:
https://github.com/BasedHardware/omi/issues/1123#issuecomment-2429615914

Reply:
Oh you are referring to the `You are an expert conversation analyzer.` part specifically, it is to improve the quality of things it extracts from the transcript